tests/IntegrationTests.py

```diff
@@ -59,9 +51,10 @@ def setUp(self):
 
     def tearDown(self):
         super(IntegrationTests, self).tearDown()
-        time.sleep(5)
-        self.server_process.terminate()
-        time.sleep(5)
+        time.sleep(2)
+        # self.server_process.terminate()
```

Comment:
Can you remove this rather than commenting it out? We have the git history if we ever want to go back to using a server process.